lambdas/evaluate-gpg45-scores/src/main/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoresHandler.java

```diff
@@ -205,6 +207,23 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private void checkCorrelation(String userId, List<VcStatusDto> currentVcStatuses)
+            throws HttpResponseExceptionWithErrorBody {
+        if (!userIdentityService.checkNameAndFamilyNameCorrelationInCredentials(
+                userId, currentVcStatuses)) {
+            LOGGER.error("Name correlation problem in user credentials for userId:{}", userId);
+            throw new HttpResponseExceptionWithErrorBody(
+                    500, ErrorResponse.FAILED_TO_NAME_CORRELATION);
+        }
+
+        if (!userIdentityService.checkBirthDateCorrelationInCredentials(
+                userId, currentVcStatuses)) {
+            LOGGER.error("Birthdate correlation problem in user credentials for userId:{}", userId);
+            throw new HttpResponseExceptionWithErrorBody(
+                    500, ErrorResponse.FAILED_TO_BIRTHDATE_CORRELATION);
```

Comment:
Could these just be `FAILED_NAME_CORRELATION` and `FAILED_BIRTHDATE_CORRELATION`